### PR TITLE
Add Ember Language Server

### PR DESCRIPTION
**Pull request by josemarluedke**

### CODE REVIEW

File: lua/lspinstall/servers/ember.lua [hunk 1]
```diff
@@ -0,0 +1,9 @@
+local config = require'lspinstall/util'.extract_config('ember')
```
Comment (kabouzeid) [inline lua/lspinstall/servers/ember.lua:1]: You can't do that, there is no 'ember' config in lspconfig.

**Review by kabouzeid — CHANGES_REQUESTED** [review]

File: lua/lspinstall/servers/ember.lua (hunk 1 above)
Comment (josemarluedke) [reply lua/lspinstall/servers/ember.lua:1]: There is one now: https://github.com/neovim/nvim-lspconfig/pull/1042

**Comment by jherdman** [comment]
Is there anything I can do to help with this PR? I'm quite new to Lua, but I'm happy to be a guinea pig to test this out.

**Comment by kabouzeid** [comment]
LGTM @jherdman please report if it's working for you :)

**Comment by jherdman** [comment]
Works! Many thanks for this change.

**Pull request merged**